StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/Link/LinkURLGeneratorTests.swift

```diff
@@ -90,6 +90,22 @@ extension STPElementsSession {
         ]
         return STPElementsSession.decodedObject(fromAPIResponse: apiResponse)!
     }
+    static func elementsSessionWithCustomerSession(apiKey: String) -> STPElementsSession {
```

Comment:
This should move to  StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetConfigurationTests.swift